migrations/2019-12-13-053745_add_crate_owner_invitation_token/down.sql

```diff
@@ -0,0 +1,3 @@
+ALTER TABLE crate_owner_invitations DROP COLUMN token;
+ALTER TABLE crate_owner_invitations DROP COLUMN token_generated_at;
+DROP FUNCTION crate_owner_invitations_set_token_generated_at();
```

Comment:
Just found another issue-- I needed to roll this migration back locally so that I could switch to a branch that doesn't have these code changes. The migration needs to drop the trigger as well, and then the changes should be undone in the reverse order in which they were done, so drop the trigger, then the function, then drop the columns.

Reply:
Yikes, good catch! I made the changes you suggested and ran the following to test:
```
diesel migration run
diesel migration redo
```
Both seemed to complete successfully.

I also ran `diesel migration revert` and manually verified that the expected columns were not present in the `crate_ownership_invitations` table, before re-running the migration. Is there anything else you'd recommend for checking the migrations?